packages/ui5-tooling-transpile/lib/util.js

```diff
@@ -112,9 +114,17 @@ async function findBabelConfigOptions(cwd) {
 	} else if (configFile) {
 		// for a babel config file we load it on our own to normalize the plugin/preset paths
 		// => no recursive merging of babel config is possible with this approach
-		babelConfigOptions = JSONC.parse(fs.readFileSync(configFile, { encoding: "utf8" }));
-		// let Babel lookup the configuration file with the Babel API
-		//partialConfig = await loadBabelConfigOptions({ configFile, filename: dir, babelrc: true });
+		try {
+			babelConfigOptions = JSONC.parse(fs.readFileSync(configFile, { encoding: "utf8" }));
+		} catch (err) {
+			// no JSON so we let Babel lookup the configuration file with the Babel API
+			const partialConfig = await babel.loadPartialConfigAsync({ cwd });
+			// but we only extract the presets and plugins and ignore the other properties
+			babelConfigOptions = {
```

Comment:
@petermuessig Is it really intended to ignore other babel config options, such as the `targets` property?
The docu of the task seems to suggest otherwise: 
>The following configuration options will only be taken into account if no inline babel configuration is maintained in the ui5.yaml as babelConfig or no external babel configuration exists in any configuration file as described in [Babels configuration section](https://babeljs.io/docs/configuration)

Reply:
Well, if I enable more, I need to rework the util.js a bit - will require a bit more time then...